chapters/interactions-nonlinear.qmd

```diff
@@ -0,0 +1,893 @@
+
+resampled_ratios %>% 
+  ggplot(aes(item_01, .estimate, col = Ordered, group = Ordered)) + 
+  geom_point() + 
+  geom_line() +
+  facet_wrap(~ Item, labeller = "label_both") +
+  geom_errorbar(aes(ymin = .lower, ymax = .upper),
+                width = 1 / 20,
+                alpha = 2 / 3) +
+  lims(y = c(25, 44)) +
+  labs(x = "Contains Item 1", y = "Mean Delivery Time") +
+  theme(legend.position = "top") + 
+  guides(col = guide_legend(title = "Contains Item 9 or 10"))
+```
+
+As a third example, interactions may occur between continuous predictors. It can be difficult to discover the interaction via visualization for two numeric predictors without converting one of the predictors to categorical bins. To illustrate the _concept_ of an interaction between two numeric predictors, $x_1$ and $x_2$, let’s use a simple linear equation:
+
+$$ 
+y = \beta_0 + \beta_{1}x_1 + \beta_{2}x_2 + \beta_{3}x_{1}x_{2} + \epsilon
+$$ {#eq-two-way-int}
+
+The $\beta$ coefficients represent the overall average response ($\beta_0$), the average rate of change for each individual predictor ($\beta_1$ and $\beta_2$), and the incremental rate of change due to the combined effect of $x_1$ and $x_2$ ($\beta_3$) that goes beyond what $x_1$ and $x_2$ can explain alone.  The parameters for this equation can be estimated using a technique such as ordinary least squares (REF).  The sign and magnitude of $\beta_3$ indicate how and the extent to which the two predictors interact:   
+
+ - When $\beta_3$ is positive, the interaction is _synergistic_ since the response increases beyond the effect of either predictor alone.  
+ - Alternatively, when $\beta_3$ is negative, the interaction is _antagonistic_ since the response decreases beyond the effect of either predictor alone.  
+ - A third scenario is when $\beta_3$ is essentially zero.  In this case, there is no interaction between the predictors and the relationship between the predictors is _additive_. 
+ - Finally, for some data sets, we may find that neither predictor is important individually (i.e., $\beta_1$ and $\beta_2$ are zero).  However, the coefficient on the interaction term is not zero.  Because this case occurs very infrequently, it is called _atypical_. 
+ 
+To understand this better, @fig-interaction-contours shows a contour plot of a predicted linear regression model with various combinations of the model slope parameters. The two predictors are centered at zero with values ranging within $x_j \pm 4.0$). The default setting shows a moderate synergistic interaction effect since all of the $\beta_j = 1.0$). In the plot, darker values indicate smaller predicted values.   
+
+::: {#fig-interaction-contours}
+
+::: {.figure-content}
+
+```{shinylive-r}
+#| label: fig-interaction-contours
+#| viewerHeight: 600
+#| standalone: true
+
+library(shiny)
+library(ggplot2)
+library(bslib)
```

Comment:
Our shiny folks helped me with a lot of formatting here. It's new syntax but it looks a lot better and will reshape itself well with very small screens.